src/document.h

```diff
@@ -724,6 +723,11 @@ struct Document {
         return GetBitmap().GetSubBitmap(r);
     }
 
+    void RefreshImageRefCount(bool includefolded = true) {

```

Comment:
Don't use default arguments unless on very frequently called functions or very exceptional arguments. Here, an explicit argument is better, it alerts the reader that a decision is being made.

Reply:
Ah thanks for the review and the good general advice.